src/pcapi/admin/custom_views/offer_view.py

```diff
@@ -81,8 +87,16 @@ def _get_venue_name(self) -> str:
         return venue.name
 
 
+def _pc_offer_url(offer_id: int) -> str:
```

Comment:
```suggestion
def _pro_offer_url(offer_id: int) -> str:
```
Je me demande si le pc apporte quelque chose. Par contre comme tu utilises le PRO_URL, voilà une proposition. A toi de voir :)